### ISSUE TITLE
Feature request: Put "tray" icons in traditional position on vertical bar right.

### ISSUE THREAD

**Issue by nextgenthemes** [material-shell/material-shell#863]
I like to have the clock and everything else that gets cramped and ugly stacked on the bottom left (or right)  where it is on all traditional DEs.

I am aware of the option to put the clock there, but that causes an ugly big bell that does not fit with the other icons to appear in its place. I really do not understand why there is an option to just move the clock and only move that and not even make it clickable for the notifications ...

Frankly, I really do not see the point of having them vertically. I am on an ultrawide, but even on a normal screen, there is plenty of space on the horizontal bar.

Furthermore, I was never really into multiple workspaces, the vertical bar's only purpose would be just for switching them, something I will probably never get around doing. Maybe I try, but it feels like a waste of scree space. Workspaces. There could just be a workspace switcher in the horizontal bar as well.

So in short, I would like a single bar to have everything. Love the auto layouts, hate the vertical bar.

**Comment by rwjack** [comment]
yeah I agree 200% with you, though @PapyElGringo probably doesn't have as much time to work on new features (or modifying current ones for that matter).

Having appindicators, power/volume, language moved to the upper right would definitely be the right move.

**Comment by nextgenthemes** [comment]
Update:

Also, I get the concept better now, and this amazing extension actually makes me really use workspaces for the first time.

But I would still like the tray moved. And the workspaces, I think they could actually be aligned horizontally and have their switcher on the horizontal bar as well. I never used more than 4 workspaces, it 3 at most for now. I have two pretty empty bars.

It's kind of cool for having them mentally stacked on top of each other and the animation, but I am not sure if it is worth an entire bar for it. I never really liked the idea of two bars. That is why I used the highly popular "Dash to Panel" rather than Ubuntu's or Gnomes default. Especially on Ultrawide, my horizontal bar is sooo damn empty.

**Comment by rwjack** [comment]
You're speaking my mind buddy :)

**Comment by abmantis** [comment]
This was the first thing I looked for in the settings after installing it :(

**Comment by jivanpal** [comment]
One practical reason for allowing the applet panel to be moved to the horizontal app bar and have a horizontal layout is that wide applets (such as CPU/RAM monitors) currently overflow in the applet panel's vertical layout.
